pubsub/azure/servicebus/queues/metadata.yaml

```diff
@@ -0,0 +1,170 @@
+      - name: azureClientId
+        required: true
+        description: "Client ID (application ID)"
+        example: '"c7dd251f-811f-4ba2-a905-acd4d3f8f08b"'
+      - name: azureClientSecret
+        sensitive: true
+        required: true
+        description: "Client secret (application password)"
+        example: '"Ecy3XG7zVZK3/vl/a2NSB+a1zXLa8RnMum/IgD0E"'
+  - title: "Azure AD: Client certificate"
+    description: "Authenticate using Azure AD with a client certificate (in PFX/PKCS#12 format). One of azureCertificate and azureCertificateFile is required."
+    metadata:
+      - name: namespaceName
+        description: "Parameter to set the address of the Service Bus namespace, as a fully-qualified domain name."
+        example: '"namespace.servicebus.windows.net"'
+        required: true
+        binding:
+          output: true
+          input: true
+      - name: azureTenantId
+        required: true
+        description: "ID of the Azure AD tenant."
+        example: '"cd4b2887-304c-47e1-b4d5-65447fdd542b"'
+      - name: azureClientId
+        required: true
+        description: "Client ID (application ID)."
+        example: '"c7dd251f-811f-4ba2-a905-acd4d3f8f08b"'
+      - name: azureCertificate
+        sensitive: true
+        required: false
+        description: "Certificate and private key (in PFX/PKCS#12 format)."
+        example: |
+          "-----BEGIN PRIVATE KEY-----\n MIIEvgI... \n -----END PRIVATE KEY----- \n -----BEGIN CERTIFICATE----- \n MIICoTC... \n -----END CERTIFICATE-----"
+      - name: azureCertificateFile
+        sensitive: true
+        required: false
+        description: "Path to PFX/PKCS#12 file on disk, containing the certificate and private key."
+        example: "/path/to/file.pem"
+      - name: azureCertificatePassword
+        sensitive: true
+        required: false
+        description: "Password for the certificate if encrypted."
+        example: "password"
+metadata:
+  - name: maxRetriableErrorsPerSec
+    description: "Maximum number of retriable errors that are processed per second. If a message fails to be processed with a retriable error, the component adds a delay before it starts processing another message, to avoid immediately re-processing messages that have failed"
+    type: number
+    default: '10'
+    example: '2'
+  - name: minConnectionRecoveryInSec
+    description: "Minimum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure."
+    type: number
+    default: '2'
+    example: '5'
+  - name: maxConnectionRecoveryInSec
+    description: "Maximum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure. After each attempt, the binding waits a random number of seconds, increasing every time, between the minimum and the maximum. Default is 300 seconds (5 minutes)."
+    type: number
+    default: '300'
+    example: '600'
+  - name: maxActiveMessages
```

Comment:
Do we also need to add `consumerID`?

Reply:
No, only in service bus topics according to latest docs.

Reply:
You were using an outdated version of the docs I believe when you reviewed this.